lg_activity/src/lg_activity/activity.py

```diff
@@ -260,6 +268,52 @@ def _is_activity_active(self):
             self.callback(self.topic, state=False, strategy='activity')
             return False
 
+    def _is_message_active(self):
+        """keep last message. Empty list returns False TODO add other cases, empty string, null, ...""" 
+        if self.messages:
+             try:
+                 self.messages = [self.messages[-1]]  # skip to last message and keep it
+                 if self.messages[0]['overlays']:  # TODO add set False methods, else True
```

Comment:
maybe do `self.messages[0].get('overlays', False)` in the case that there's no overlays